CMS-Enterprise/sbom-harbor
# chore: stop running cargo test on commit

## Summary

To save time, we are not going to be running the tests on each commit anymore.   

### Changed

Removed the configuration in .pre-commit-config.yaml that ran cargo test on commit.

## How to test

Change a file and commit it.  Observe that `cargo test` was _not_ run:

```
fmt......................................................................Passed
cargo clippy.............................................................Passed
cargo check..............................................................Passed
check json...........................................(no files to check)Skipped
check yaml...........................................(no files to check)Skipped
check for case conflicts.................................................Passed
check for merge conflicts................................................Passed
check for broken symlinks............................(no files to check)Skipped
detect destroyed symlinks................................................Passed
fix utf-8 byte order marker..............................................Passed
fix end of files.........................................................Passed
trim trailing whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing core/src/lib.rs

mixed line ending........................................................Passed
detect aws credentials...................................................Passed
detect private key.......................................................Passed
Detect hardcoded secrets.................................................Passed
This will not be there===> Cargo test...............................Passed
```
